features/step_definitions/evidence_steps.rb

```diff
@@ -110,10 +110,10 @@
 Given("I have successfully submitted the evidence") do
   click_on 'Start now', visible: false
   evidence_accuracy_page.content.correct_evidence.click
-  click_on 'Next', visible: false
+  next_page
   fill_in 'Total monthly income from evidence', with: '500'
-  click_on 'Next', visible: false
-  click_on 'Next', visible: false
+  next_page
+  next_page
```

Comment:
shouldn't we check what's on that page to give capybara time to load the page?

Reply:
Good point Petr, I'll add something